python/pyspark/sql/utils.py

```diff
@@ -112,6 +112,15 @@ def toJArray(gateway, jtype, arr):
     return jarr
 
 
+def _require_minimum_pandas_version():
```

Comment:
Ohh. BTW, I think we can just name it `_require_minimum_pandas_version` -> `require_minimum_pandas_version` and `_require_minimum_pyarrow_version` -> `require_minimum_pyarrow_version`. It's fine as is too to me.

Reply:
Sounds good. I'll update them. Thanks!